lib/Mojolicious/Plugin/WebSocketProxy.pm

```diff
@@ -14,6 +14,10 @@ sub register {
 
     die 'No base path found!' unless $config->{base_path};
 
+    my $localizer = (ref($config->{localizer}) // '?') eq 'CODE'
```

Comment:
no need for the `//` fallback here, `ref` returns an empty string for non-refs/undef. However, I'd suggest better to make it `$config->{localizer} || sub { $_[1] }` otherwise it's silently ignoring invalid parameters such as `localizer => "some_helper_name"`.